service/src/test/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationServiceTest.java

```diff
@@ -159,8 +158,8 @@ public void getMutationAssessorEnrichedAnnotationByVariantString()
         this.mockMutationAssessorServiceMethods(variantMockData, maMockData);
         this.mockEnsemblServiceMethods();
 
-        List<String> fields = new ArrayList<>(1);
-        fields.add("mutation_assessor");
+        List<AnnotationField> fields = new ArrayList<>(1);
+        fields.add(AnnotationField.MUTATION_ASSESSOR);
```

Comment:
`AnnotationField.MUTATION_ASSESSOR` actually returns an all uppercase string value  `MUTATION_ASSESSOR`. Let's make sure that we can still handle lower case values (e.g: `mutation_assessor`) for those fields.